content/departments/cloud/technical-docs/operations.md

```diff
@@ -15,9 +15,9 @@ Operations guides for [managed instances](./index.md).
 To perform any MI opertaions, you need to meet the following requirement
 
 ```sh
-git clone https://github.com/sourcegraph/deploy-sourcegraph-managed
+git clone git@github.com:sourcegraph/deploy-sourcegraph-managed.git
```

Comment:
Because:
```
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/sourcegraph/deploy-sourcegraph-managed.git/'

```